src/main/resources/reference.conf

```diff
@@ -170,5 +170,8 @@ zap-automation-config {
 
     # Interval value for patience config in seconds. Spider and Active Scan status is checked at this interval until the scan is completed.
     patienceConfigInterval=2
+
+    # Zap connection timeout in seconds. Zap will timeout if a response is not received within the configured duration.
+    connectionTimeout=20
```

Comment:
I would normally let teams specify time with strings such as "1 second", "2 minutes" which is automatically parsed to a Duration object if you use typesafe config. Alternatively I'd do "connectionTimeoutInSeconds" to make it explicit. But we have `patienceConfigTimeout` already set to 600 and connectionTimeout is consistent with that so leave as you did may be less confusing for teams.